libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2359,7 +2359,18 @@ bool TypeChecker::visit(Identifier const& _identifier)
 				SecondarySourceLocation ssl;
 
 				for (Declaration const* declaration: annotation.overloadedDeclarations)
-					ssl.append("Candidate:", declaration->location());
+					if (declaration->location().isEmpty())
+					{
+						// Try to re-construct function definition
+						string description;
+						for (auto const& param: declaration->functionType(true)->parameterTypes())
+							description += (description.empty() ? "" : ", ") + param->toString(false);
+						description = "function " + _identifier.name() + "(" + description + ")";
```

Comment:
Do we need ``function ``? Or do we want to use ``builtin ``? Hm - ``function `` is fine I guess.